src/components/DotNavigation/docs/index.story.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from 'react';
+import { DotNavigation, Theme } from '../DotNavigation';
+import * as examples from './examples';
+import {
+  header,
+  api,
+  divider,
+  importExample,
+  playground,
+  tab,
+  code as baseCode,
+  tabs,
+  testkit,
+  title,
+  description,
+} from 'wix-storybook-utils/Sections';
+import { settingsPanel } from '../../../../stories/utils/SettingsPanel';
+import { allComponents } from '../../../../stories/utils/allComponents';
+import * as ExtendedRawSource from '!raw-loader!./DotNavigationExtendedExample.tsx';
+import * as ExtendedCSSRawSource from '!raw-loader!./DotNavigationExtendedExample.st.css';
+import { DotNavigationExtendedExample } from './DotNavigationExtendedExample';
+import * as Readme from '../README.md';
+
+const code = config =>
+  baseCode({ components: allComponents, compact: true, ...config });
+
+export default {
+  category: 'Components',
+  storyName: 'DotNavigation',
+  component: DotNavigation,
+  componentPath: '../DotNavigation.tsx',
+  componentProps: () => ({
+    'data-hook': 'storybook-DotNavigation',
+    onSelect: (index: number) => console.log(index),
+  }),
+  exampleProps: {
+    theme: Object.values(Theme),
+    onSelect: [],
+  },
+  dataHook: 'storybook-DotNavigation',
+  sections: [
+    header(),
+    tabs([
+      tab({
+        title: 'Usage',
+        sections: [
+          description(Readme),
+
+          importExample({
+            source: examples.importExample,
+          }),
+
+          divider(),
+
+          title('Examples'),
+
+          ...[
+            { title: 'Default Example', source: examples.example },
+            { title: 'Three Dot Example', source: examples.threeDotExamle },
+            {
+              title: 'Ten Dot Example',
+              source: examples.tenDotExamle,
+            },
+            { title: 'Light Example', source: examples.lightExample },
+            {
+              title: 'Light Example with Safety Border',
+              source: examples.lightExampleWithSafetyBorder,
+            },
+            {
+              title: 'Default Example with currentIndex = 2',
+              source: examples.defaultExampleWithCurrentIndex,
+            },
+          ].map(code),
+        ],
+      }),
+
+      ...[
+        { title: 'API', sections: [api()] },
+        { title: 'TestKit', sections: [testkit()] },
+        { title: 'Playground', sections: [playground()] },
+        {
+          title: 'Settings Panel',
+          sections: [
+            settingsPanel({
+              example: (
+                <DotNavigationExtendedExample
+                  length={7}
+                  showBorder
+                  theme={Theme.Light}
```

Comment:
Since the background of the example is white, it might be better to start with the dark theme.

Reply:
The thing is that default SelectedBorderColor for the dark theme is color-1, it is the same as the color of every palette background, so you cannot see the border by default unless you change its color in the palette.

If you are ok with this, I'll remove the light theme from here.